main.py

```diff
@@ -465,6 +468,14 @@ def main():
     parser.add_argument("--pte", type=int)
     parser.add_argument("--d", type=int)
     parser.add_argument("--whitening", type=int, default=1)
+    parser.add_argument("--data_param1", type=int, default=None, help=
```

Comment:
C'est déjà `None` par default 
```suggestion
    parser.add_argument("--data_param1", type=int, help=
```

C'est toujours un int ? Peu être que mettrait un float, par exemple `stretching` ne peut pas être float ?
```suggestion
    parser.add_argument("--data_param1", type=float, default=None, help=
```
Et puis quand tu récupère tu fais `bins = int(param[0])`

Reply:
oui, bien sur, j'ai réalisé !